SQLscripts/README.md

```diff
@@ -0,0 +1,8 @@
+# Zenodo spam classifiers - Dataset dump 
```

Comment:
- I prefer naming this folder `scripts`, just for consistency with the other single-word lower-case folders. 
- Also we might add in the future other "administrative" scripts in this folder (for publishing the models, deploying, etc.) which will not be SQL only, but probably shell scripts